experimental/auth-logic/wrappers/rekor_wrapper.go

```diff
@@ -186,6 +187,14 @@ func checkEntryPubKeyMatchesExpectedKey(rekordEntry *rekord.V001Entry, prodTeamK
 	return nil
 }
 
+func compareEndorsementAndRekordHash(rekordEntry *rekord.V001Entry, endorsementBytes []byte) error {
+	endorsementHash := fmt.Sprintf("%x", sha256.Sum256(endorsementBytes))
+	if endorsementHash != *rekordEntry.RekordObj.Data.Hash.Value {
+		return fmt.Errorf("Hash values of endorsement bytes and rekor dentry not equal. endorsementBytes: %v, endorsementHash: %s, rekordEntry: %v, rekordHash: %v", endorsementBytes, endorsementHash, rekordEntry, *rekordEntry.RekordObj.Data.Hash.Value)
```

Comment:
The error message looks very verbose. Perhaps just print the expected hash and the actual one with the `got/want` format.

Reply:
OK I made it just include the two hashes. I thought `got/want` should be just for tests, so I left it saying `endorsement hash: %v, rekor hash: %v`, since if I were reading the message I wouldn't be sure which one is got and which one is want.

If you still would rather it be got/want, which should be "got"? The endorsement one?

Reply:
Makes sense. Thanks. I don't particularly like the `got/want` style, but I think I've seen it used in code other than the tests (I could be wrong). Your wording is clearer in my opinion :)